components/breadcrumb/BreadcrumbItem.razor

```diff
@@ -5,17 +5,22 @@
 
     @if (Overlay != null)
     {
-        <span class="ant-breadcrumb-overlay-link" nz-dropdown [nzDropdownMenu]="nzOverlay">
-            <span class="ant-breadcrumb-link">
-                @ChildContent
-            </span>
-            <Icon Type="down"></Icon>
-        </span>
+        <Dropdown>
+            <ChildContent>
```

Comment:
We recommend using the unbound method to bind the trigger. And when we use it internally we need to use unbuound.

Reply:
Can you show an example with using `Unbound` property? I can't find a working example.

Reply:
```
<Dropdown>
    <Overlay>
     ...
    </Overlay>
    <Unbound>
        <div @ref="@context.Current">
         Trigger
        </a>
    </Unbound>
</Dropdown>
```